crates/llm-remote-http/src/lib.rs

```diff
@@ -0,0 +1,186 @@
+use anyhow::Result;
+use reqwest::{
+    header::{HeaderMap, HeaderValue},
+    Client, Url,
+};
+use serde::{Deserialize, Serialize};
+use serde_json::json;
+use spin_core::async_trait;
+use spin_llm::{access_denied_error, LlmEngine};
+use spin_world::llm::{self as wasi_llm, InferencingParams};
```

Comment:
nit: why not just also prefix `InferencingParams` like everything else from `wasi_llm`?. In fact, since there's nothing else called `llm` here we could probably just import this as `llm`, no?

Reply:
I kept it consistent across the codebase, I will change it to have the `wasi_llm` prefix.